.travis.yml

```diff
@@ -32,11 +31,21 @@ matrix:
   - name: Coding Standards
     php: 7.3
     env: WP_TRAVISCI=phpcs
+  - name: Static Analysis
+    php: 7.4
+    env: WP_TRAVISCI=phpstan
+  allow_failures:
```

Comment:
Should we maybe add the phpcs to allow_failures as well, since we're basically just importing this package? Or maybe remove the formatting entries in the matrix altogether?